doc/conf.py

```diff
@@ -13,7 +13,8 @@
 author = "Ploomber"
 comments_config = {"hypothesis": False, "utterances": False}
 copyright = "2023"
-exclude_patterns = ["**.ipynb_checkpoints", ".DS_Store", "Thumbs.db", "_build"]
+exclude_patterns = ["**.ipynb_checkpoints", ".DS_Store",
+                    "Thumbs.db", "_build", "user-guide/table_explorer_demo.ipynb"]
```

Comment:
oh, I see there are two files. one md and one ipynb. I guessing the demo is temporary? if so, let's delete it from this repository so we don't merge it (I already have a copy)

Reply:
also, I think this section got renamed to `nb_execution_excludepattern` it still works probably due to backward compatibility, but let's merge it with the other one (line 18)

Reply:
so there are two observations here:

1. migrate exclude_patterns to nb_execution_excludepattern since jupyter-book renamed the variable
2. I see we have table_explorer.md and table_explorer_demo.ipynb, do we need both files? seems like we only need the .md file. the .md file can contain the screenshot that's fine and then the binder link so people try it

Reply:
The binder link opens `table_explorer_demo.ipynb`. Do you think we should open our Binder instead?

Reply:
can't we merge table_explorer_demo.ipynb into table_explorer.md so we keep a single file?

Reply:
Done. 
I removed the image since I noticed that if we use `:tags: []` it draws the table so it looks better.

[example](https://jupysql--420.org.readthedocs.build/en/420/user-guide/table_explorer.html#create-the-table)